src/pybind/mgr/dashboard/services/orchestrator.py

```diff
@@ -200,6 +200,13 @@ def stop(self) -> str:
         return self.api.upgrade_stop()
 
 
+class HardwareManager(ResourceManager):
+
+    @wait_api_result
+    def common(self, category: str, hostname: Optional[List[str]] = None) -> str:
```

Comment:
?
```suggestion
    def summary(self, category: str, hostname: Optional[List[str]] = None) -> str:
```

Reply:
I left it as `common` since it calls `node_proxy_common` which calls `common`,  the `NodeProxyCache` class already has defined a method called `summary` which I think could end up being confusing calling this as `summary`